c++11/src/logger.cpp

```diff
@@ -0,0 +1,11 @@
+#include "logger.h"
+
+namespace lightstep {
+//------------------------------------------------------------------------------
+// GetLogger
+//------------------------------------------------------------------------------
+spdlog::logger& GetLogger() {
+  static auto logger = spdlog::stderr_logger_mt("lightstep");
```

Comment:
Yes, iostream would probably be sufficient. I put it in mostly so that it could match the [configurable logging](https://github.com/lightstep/lightstep-tracer-go/blob/master/options.go#L123) that the go tracer has, but it should be easy to remove.

Envoy also uses [spdlog](https://github.com/lyft/envoy/blob/master/source/common/common/logger.h#L12) btw.